ceph-perf-pull-requests/config/definitions/ceph-perf-pull-requests.yml

```diff
@@ -62,6 +62,7 @@
               if type -t clang-$i > /dev/null; then
                   cxx_compiler="clang++-$i"
                   c_compiler="clang-$i"
+                  export CFLAGS=$(echo $CFLAGS | (sed -e 's/\-fstack-clash-protection//'))
```

Comment:
Where the `-fstack-clash-protection` is getting injected to our `CFLAGS`? Maybe it's worth to differentiate there instead of filtering here?

Reply:
yes, but seems like it is coming from system values, we are setting `CFLAGS` & `CXXFLAGS` to $(RPM_OPT_FLAGS)  + extra args we can drop using RPM_OPT_FLAGS and can use custom input, but should we?
also, should update CXXFLAGS if we go with this approach...
```
The variable RPM_OPT_FLAGS is set using values in /usr/lib/rpm/rpmrc. Look there to make sure you are using values appropriate for your system (in most cases you are). Or simply don't use this variable in your spec file. It is optional.
```